pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 120
```

Comment:
De même pour ce fichier local de config.
Si on se met tous d'accord d'utiliser Black et Flake8 avec les mêmes config, why not

Reply:
D'accord avec toi. Tu utilises Black aussi non ? ça fait sens de ne plus avoir de débat sur le formatting avec les pre-commit hooks.

Reply:
Yes pour ma part j'utilise pre-commit avec black 120 caractères et flake8 120 caractères avec quelques warnings de formatting ignorés

Reply:
Bon bah on peut se mettre d'accord là dessus et profiter de la PR d'Ali pour l'ajouter ?

Reply:
Moi ça me va 👍